src/pages/documentation/release-notes/index.md

```diff
@@ -129,6 +131,8 @@ Note that all these extensions must be updated together. Flutter and React plugi
 
 * Major version update for [Adobe Campaign Classic workflows](./index.md) for Adobe Experience Platform Mobile SDKs on Android compatible with Mobile Core 2.0.0. This library is now available as an [open source project on GitHub](https://github.com/adobe/aepsdk-campaignclassic-android).
 
+Please note `registerDevice` API, similar to iOS, no longer provides a callback method to provide boolean registration status as a `false` value cannot be accurately used as a signal for achieving success upon retries.
```

Comment:
"Please note that the `registerDevice` API, similar to iOS, no longer provides a callback method to provide a boolean registration status since a `value` value cannot..."